guide/creating-your-bot/commands-with-user-input.md

```diff
@@ -173,7 +173,7 @@ if (!message.mentions.users.size) {
 ```
 
 ::: tip
-If you're wondering what `message.reply()` does, it's just an alternative for `message.channel.send()` which also prepends a mention of the person who sent the message, unless used in a DM. It can be very useful for providing feedback!
+If you're wondering what `message.reply()` does, it's just an alternative for `message.channel.send()`, which also prepends a mention of the person who sent the message, unless used in a DM. It can be very useful for providing feedback!
```

Comment:
```suggestion
`message.reply()` is alternative to `message.channel.send()` that prepends a mention of the person who sent the message, unless the message was sent in a DM. It can be very useful for providing feedback!
```